src/dialog/Dialog.ts

```diff
@@ -65,6 +67,24 @@ export default class Dialog extends DialogBase<DialogProperties> {
 		!this.properties.modal && this._onCloseClick();
 	}
 
+	private _onKeyUp(event: KeyboardEvent) {
+		const { keyCode } = event;
```

Comment:
You could either destructure the argument itself to save two lines (`private _onKeyUp({ keyCode }) {`) or you could really not even destructure at all since you only ever reference a property on `event` once, so it's equally as optimal to just use `event.keyCode` directly. That'd be my vote, not to over-destructure.

Reply:
happy to do so.